portfolio/src/main/webapp/script.js

```diff
@@ -91,47 +94,100 @@ function scrollToElement(elementId, duration = 300)
                 }, duration);
 }
 
+/**
+ * Function that inits the comments section
+ * Loads the comments languages available in a dropdown
+ * Then load the comments in that language
+ */
+function initComments() {
+  // Make a GET request to '/languages' and parse the response json into 'languages' array
+  fetch('/languages').then(response => response.json()).then((languages) => {
+    // Get the comments language element
+    const commentsLanguageElement = document.getElementById('comments-language');
+
+    // For each language create and add an option to the comments language element
+    for(let language of languages) {
+      const optionElement = document.createElement('option');
+      optionElement.text = language.displayName + ' (' + language.code + ')';
+      optionElement.value = language.code;
+      commentsLanguageElement.add(optionElement);
+    }
+
+    // If the user didn't choose other language before, set it to 'original'
+    if(localStorage.hasOwnProperty('commentsLanguageCode') === false)
```

Comment:
Typically, comparison to 'true' or 'false' shouldn't be explicit. Is it necessary in this case?

Reply:
Changed it to ` if(!localStorage.hasOwnProperty('commentsLanguageCode'))` , thanks!